imgui-knobs.cpp

```diff
@@ -42,7 +42,12 @@ namespace ImGuiKnobs {
                  float _angle_min,
                  float _angle_max) {
                 radius = _radius;
-                t = ((float) *p_value - v_min) / (v_max - v_min);
+                if (flags & ImGuiKnobFlags_Logarithmic) {
+                    float v = std::max(std::min(*p_value, v_max), v_min);
```

Comment:
`std::max` needs `#include <algorithm>`, please use `ImMax`/`ImMin`

Reply:
fixed in 7d56d77cdf896b6985c3a078393c9ad77c86327a